### ISSUE TITLE
Question about Notion Support?

### ISSUE THREAD

**Issue by xlfrog** [AI-Citizen/SolidGPT#86]
In the previous version I had the capability to upload output to Notion and sync it back seamlessly. I'm wondering whether this functionality is still supported in the current version.
Additionally, I'm interested in knowing if there are plans to further focus on and enhance Notion integration in future updates.

**Comment by wwwwww1020** [comment]
Yes, our product still support using with Notion, you can check our v0.1.0 intro for the details.https://github.com/AI-Citizen/SolidGPT/blob/main/docs/READMEv1.md
In the future our target would be make SolidGPT as the "Notion" of Software Engineering, we will publish more details in the future.

**Comment by xlfrog** [comment]
Can I use your new web UI to communicate with my Notion now?
